library/pluginsystem/PluginSystem.php

```diff
@@ -76,7 +76,7 @@ public function init()
         $composerJsonPath = realpath( __DIR__."/../../composer.json" );
         $composerJson = file_get_contents( $composerJsonPath );
         if ( $composerJson === false ) {
-            throw new Exception( "$composerJsonPath/composer.json is not readable" );
+            throw new \Exception( "$composerJsonPath/composer.json is not readable" );
```

Comment:
Yes, a distinction with a difference I suppose.  Still scary when if dawned on me.  Scope and namespace have always been interchangeable terms with me, and I have mental issues with scope in web scripting.  IF loops don't seem to contain a local variable to it's scope.  Globals are not accessible in child functions, and now there is a php namespace without fine-grain scope control...all very weird.  I will shut up now.

Does this tie into the fancy appointment stuff going on elsewhere?